admin/server.go

```diff
@@ -89,7 +116,11 @@ func (s *Server) Start() {
 	rs.RegisterCodec(json.NewCodec(), "application/json")
 	rs.RegisterCodec(json.NewCodec(), "application/json;charset=UTF-8")
 
-	rs.RegisterService(s.nodeWallet, "")
+	if s.nodeWallet != nil {
```

Comment:
I think we have a IsValidator call we could use instead? not a big deal, but the nodewallet would be instatiated evern for a non-validators isn't it?